server/db/schema/domain.ts

```diff
@@ -13,8 +13,8 @@ export const domains = pgTable("domain", {
 		.$defaultFn(() => nanoid()),
 	host: text("host").notNull(),
 	https: boolean("https").notNull().default(false),
-	port: integer("port").default(80),
-	path: text("path").default("/"),
+	port: integer("port").default(80).notNull(),
+	path: text("path").default("/").notNull(),
```

Comment:
What's the reason on change this? by default they have a default value so is more safe and it is working fine, I think if there is any case that could be too rare to have null values in these fields the migration would not work.

Reply:
The [domain](https://github.com/Dokploy/dokploy/pull/267/files#diff-dcdfd5c7c1b7af40e1baddcf77ed5eaa6a7c34133de9b1b63199920e1b602373R38) type was throwing an error. I added the nullable modifier there, and it's fixed now.